### DISCUSSION TITLE
DAI 2024-2025 - My Docker Compose application - Killian Viquerat

### DISCUSSION THREAD

**Discussion by Killian-Viquerat** [heig-vd-dai-course/heig-vd-dai-course#414] [Show and tell]
![image](https://github.com/user-attachments/assets/bf064181-621d-47da-9686-c84ba87ecf1c)

https://github.com/Killian-Viquerat/DAI-06-DockerJavaIos

**Comment by ludelafo** [comment]
Thank you!

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
